### ISSUE TITLE
Fails to build on 0.25.0

### ISSUE THREAD

**Issue by bararchy** [veelenga/ameba#64]
```crystal
in src/ameba/cli/cmd.cr:10: instantiating 'Ameba::Config:Class#load(String)'

    config = Config.load opts.config
                    ^~~~

in src/ameba/config.cr:48: instantiating 'Ameba::Config:Class#new(YAML::Any)'

    Config.new YAML.parse(content)
           ^~~

in src/ameba/config.cr:33: instance variable '@rules' of Ameba::Config must be Array(Ameba::Rule::Base), not Array(Ameba::Rule::ComparisonToBoolean | Ameba::Rule::ConstantNames | Ameba::Rule::DebuggerStatement | Ameba::Rule::EmptyEnsure | Ameba::Rule::EmptyExpression | Ameba::Rule::HashDuplicatedKey | Ameba::Rule::LargeNumbers | Ameba::Rule::LineLength | Ameba::Rule::LiteralInCondition | Ameba::Rule::LiteralInInterpolation | Ameba::Rule::MethodNames | Ameba::Rule::NegatedConditionsInUnless | Ameba::Rule::PercentArrays | Ameba::Rule::PredicateName | Ameba::Rule::RandZero | Ameba::Rule::RedundantBegin | Ameba::Rule::ShadowedException | Ameba::Rule::Syntax | Ameba::Rule::TrailingBlankLines | Ameba::Rule::TrailingWhitespace | Ameba::Rule::TypeNames | Ameba::Rule::UnlessElse | Ameba::Rule::UnneededDisableDirective | Ameba::Rule::UnusedArgument | Ameba::Rule::UselessAssign | Ameba::Rule::UselessConditionInWhen | Ameba::Rule::VariableNames | Ameba::Rule::WhileTrue)

    @rules = Rule.rules.map &.new(config)
    ^~~~~~

```

**Comment by veelenga** [comment]
Did you try 0.7.0?

**Comment by bararchy** [comment]
Fixed at `0.7.0` my bad
